tool/templates/kotlin/DiplomatStr.java.jinja

```diff
@@ -0,0 +1,16 @@
+package {{domain}}.{{lib_name}};
```

Comment:
would it make sense to package this and related fundamentals as a standalone dependency?

Reply:
we've not been doing that so far because we're trying to be self contained. not opposed, but i think for now it's fine to just package and duplicate

Reply:
👍